app/models/incomeSourceDetails/viewmodels/CheckDetailsViewModel.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.incomeSourceDetails.viewmodels
+
+import enums.IncomeSourceJourney.IncomeSourceType
+
+import java.time.LocalDate
+
+
+case class CheckDetailsViewModel(businessName: Option[String] = None,
+                                 businessStartDate: Option[LocalDate],
+                                 accountingPeriodEndDate: Option[LocalDate] = None,
+                                 businessTrade: Option[String] = None,
+                                 businessAddressLine1: Option[String] = None,
+                                 businessAddressLine2: Option[String] = None,
+                                 businessAddressLine3: Option[String] = None,
+                                 businessAddressLine4: Option[String] = None,
+                                 businessPostalCode: Option[String] = None,
+                                 businessCountryCode: Option[String] = None,
+                                 incomeSourcesAccountingMethod: Option[String] = None,
+                                 cashOrAccrualsFlag: String,
+                                 showedAccountingMethod: Boolean = true,
+                                 incomeSourceType: IncomeSourceType
+ ){
+
+  def countryName: Option[String] = Some("United Kingdom")
```

Comment:
I would assume we should point to relevant language resource here, in the same fashion as below

Reply:
We don't have a message for this, or the Welsh. We would need to ask Olga to request the Welsh. Maybe best to leave to another ticket?

Reply:
as long as this is raised/communicated, should be fine